README.md

```diff
@@ -207,7 +207,7 @@ client = AsyncAI21Client(
 async def main():
     response = await client.chat.completions.create(
         messages=messages,
-        model="jamba-instruct-preview",
+        model="jamba-1.5",
```

Comment:
Maybe its worth mixing it up and adding `jamba-1.5-large` to some of the snippets as well. So some examples would have `jamba-1.5` and some would have `jamba-1.5-large`. wdyt?